addons/website/static/src/scss/website.scss

```diff
@@ -1780,8 +1780,7 @@ input[value*="data-oe-translation-initial-sha"] {
     }
 }
 
-// Offcanvas menu for in page filters (e.g. product list)
-.o_website_offcanvas .accordion-button {
+.accordion-button {
```

Comment:
I'm not sure to understand this change, or maybe I missed something ?

Reply:
@stefanorigano do you have an idea on why this is needed ?

Reply:
After discussions we decided to remove completely this piece of code to reset to bootstrap default